### PR TITLE
[PPML] Fix wrong word in QuickStart doc

### PR DESCRIPTION
**Pull request by Uxito-Ada**
This is the fix for issue #3586

### CODE REVIEW

**Comment by hkvision** [comment]
There are also elsewhere in the document pages that have the typo, need to have a thorough search and modify all of them.

**Comment by Uxito-Ada** [comment]
Fix all typo in the repo now.

**Review by hkvision — APPROVED** [review]

**Comment by hkvision** [comment]
Thanks for the fix.

**Comment by qiyuangong** [comment]
@Uxito-Ada Need to change https://github.com/intel-analytics/BigDL/blob/branch-2.0/docs/readthedocs/source/index.rst after rename doc title

File: ppml/trusted-realtime-ml/scala/docker-graphene/kubernetes/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Trusted Realtime ML for Kubernetes
 
-## Deploy the Intel SGX Device Plugin for Kubenetes
+## Deploy the Intel SGX Device Plugin for Kubernetes
 
 Please refer to the document [here][devicePluginK8sQuickStart].
```
Comment (qiyuangong) [inline ppml/trusted-realtime-ml/scala/docker-graphene/kubernetes/README.md:6]: Line 74 https://bigdl.readthedocs.io/en/latest/doc/PPML/QuickStart/deploy_intel_sgx_device_plugin_for_kubernets.html also need be modified

**Review by qiyuangong — APPROVED** [review]
LGTM

**Pull request merged**